fetch.bs

```diff
@@ -5313,6 +5335,10 @@ run these steps:
    <li><p><a abstract-op lt="append the Fetch metadata headers for a request">Append the Fetch metadata headers for <var>httpRequest</var></a>.
    [[!FETCH-METADATA]]
 
+   <li><p>If <var>httpRequest</var>'s <a for=request>initiator</a> is "<code>prefetch</code>", then
+   <a>set a structured field value</a> given (`<a http-header><code>Sec-Purpose</code></a>`,
+   "<code>prefetch</code>") in <var>httpRequest</var>'s <a for=request>header list</a>.
```

Comment:
We're hitting a problem here with this API and ambiguity between strings and tokens.

We could omit the quotes, but that's not really a thing we normally define. Perhaps we just write it out as:

> token "`prefetch`"

with an xref for "token"?

@domenic thoughts?

Reply:
I would do

> the token `prefetch`

I think it's confusing to have both "token" as the marker (indicating a token), and the quotation marks as the marker (indicating a string).